### PR TITLE
33 admin invoice show page

### PR DESCRIPTION
**Pull request by JCBeldo**
This PR completes US 33 and closes issue #25

### CODE REVIEW

File: app/models/customer.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Customer < ApplicationRecord
+  self.primary_key = :id
+  has_many :invoices
+  has_many :invoice_items, through: :invoices
+  has_many :items, through: :invoice_items
+  has_many :merchants, through: :items
+  has_many :transactions, through: :invoices
+
+  def self.top_5_customers
```
Comment (garrettgregor) [inline app/models/customer.rb:9]: 💥

File: app/views/admin/_admin_nav.html.erb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<h1 style="float:left; margin:20px;">Admin Dashboard</h1>
+<h3 style="float:right; background-color:lightgrey; margin:20px;"><%= link_to "Dashboard", admin_index_path, style: "padding:10px; border-style: outset;" %>
```
Comment (garrettgregor) [inline app/views/admin/_admin_nav.html.erb:2]: I like that you added some additional formatting here! This is gonna be so pretty 🤩

File: app/views/admin/invoices/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<%= render partial: "admin/admin_nav" %>
+
+<br><br>
+
+<br><br><h2 style="background-color:lightgrey; border-top: 3px solid black; padding:10px; margin:20px;"><center>Invoices</h2>
```
Comment (garrettgregor) [inline app/views/admin/invoices/index.html.erb:5]: I didn't know you could chain the `<br>`'s here

File: app/models/invoice.rb [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  enum :status, ["cancelled", "in progress", "completed"]
+
+  def self.sorted_incomplete_invoices
+    Invoice.joins(:invoice_items)
+    .where("invoice_items.status !=2")
+    .group(:id)
+    .order(:created_at)
+  end
+
+  def format_time_stamp
```
Comment (garrettgregor) [inline app/models/invoice.rb:17]: this can be taken out...

**Review by garrettgregor — CHANGES_REQUESTED** [review]

File: app/views/admin/invoices/index.html.erb (hunk 3 above)
Comment (JCBeldo) [reply app/views/admin/invoices/index.html.erb:5]: I found out last night and it works a charm.

File: app/models/invoice.rb (hunk 4 above)
Comment (JCBeldo) [reply app/models/invoice.rb]: done :)

**Review by garrettgregor — APPROVED** [review]

**Review by garrettgregor — APPROVED** [review]
Interviewer: "How have you ever used inheritance in your applications?"
Self: {links repo}

**Pull request merged**